libnetwork/network/netavark_interface.go

```diff
@@ -0,0 +1,47 @@
+//go:build linux || freebsd
```

Comment:
I think this file should just go into interface.go, since it's built regardless of CNI